### ISSUE TITLE
frozen window

### ISSUE THREAD

**Issue by DilaraSina** [victordibia/handtracking#39]
Hi, I get below error and display window is frozen (as soon as implementation starts).

> ResourceExhaustedError (see above for traceback): OOM when allocating tensor of shape [1024] and type float
	 [[Node: FeatureExtractor/MobilenetV1/Conv2d_13_pointwise/BatchNorm/gamma = Const[dtype=DT_FLOAT, value=Tensor<type: float shape: [1024] values: 6.0167079 5.9263978 6.43220091...>, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

>> loading frozen model for worker
> ====== loading HAND frozen graph into memory
2019-02-22 16:47:36.197699: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-02-22 16:47:36.261777: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-02-22 16:47:36.262137: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 0 with properties: 
name: GeForce GTX 1060 with Max-Q Design major: 6 minor: 1 memoryClockRate(GHz): 1.48
pciBusID: 0000:01:00.0
totalMemory: 5.94GiB freeMemory: 44.25MiB
2019-02-22 16:47:36.262154: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1060 with Max-Q Design, pci bus id: 0000:01:00.0, compute capability: 6.1)
2019-02-22 16:47:36.262774: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 44.25M (46399488 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
2019-02-22 16:47:36.263103: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 39.83M (41759744 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
>  ====== Hand Inference graph loaded.
2019-02-22 16:47:36.269881: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1060 with Max-Q Design, pci bus id: 0000:01:00.0, compute capability: 6.1)

**Comment by DilaraSina** [comment]
should I run the code by using regular tensorflow rather than tensorflow-gpu?

**Comment by victordibia** [comment]
The error above suggests you dont have enough memory (on GPU) to run this process. 
You typically do not need a gpu for decent inference ...
See issue #26 .

-V.
